Added readme file (#3)
rakanshaker/Color-Palette-Generator: readme.md

> Welcome to my Color Palette Generator!
> 
> This is a simple webapp that generates random colors, allows users to lock or copy hex values, and enter their own colors for testing purposes.
> 
> To run, open "index.html" in your web browser.

Reviewer: What's cool about markdown is you can format the text in **all** *sorts* ~~of ways~~. For example, its arguably clearer to quote code or a file name like `index.html` instead of "index.html". Here is [some documentation](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) on the things you can do.

```suggestion
# Color Palette Generator
### About the Project
Welcome to my Color Palette Generator! This is a simple web app that generates random colors, allows users to lock or copy hex values, and enter their own colors for testing purposes.
### How to Run
To run, simply open `index.html` in your web browser.
```
